fe/fe-core/src/main/java/org/apache/doris/catalog/PartitionKey.java

```diff
@@ -59,9 +60,13 @@
 
 public class PartitionKey implements Comparable<PartitionKey>, Writable {
     private static final Logger LOG = LogManager.getLogger(PartitionKey.class);
+    @SerializedName("ks")
     private List<LiteralExpr> keys;
```

Comment:
Done

Reply:
Hello, if the base class of "LiteralExpr", "Expr", has already registered a RuntimeTypeAdaptor containing "LiteralExpr"'s subtypes, should I register another RuntimeTypeAdaptor to avoid the deserialization failure? Because I noticed that a subtype registered to two different RuntimeTypeAdaptors will cause UT error "com.google.gson.JsonParseException: cannot serialize org.apache.doris.analysis.IntLiteral because it already defines a field named clazz"